HW_1/task_3.py

```diff
@@ -0,0 +1,15 @@
+# Задача 6: Вы пользуетесь общественным транспортом?
+# Вероятно, вы расплачивались за проезд и получали билет с номером.
+# Счастливым билетом называют такой билет с шестизначным номером,
+# где сумма первых трех цифр равна сумме последних трех. Т.е. билет с номером 385916 – счастливый,
+# т.к. 3+8+5=9+1+6. Вам требуется написать программу, которая проверяет счастливость билета.
+# *Пример:*
+# 385916 -> yes
+# 123456 -> no
+
+n = int(input('Введите шестизначное число: '))
+a = n // 1000
+b = n % 1000
+if (a % 10 + (a // 10) % 10 + a // 100) == (b % 10 + (b // 10) % 10 + b // 100): print('YES')
+else: print('NO')
```

Comment:
Во-первых, опять лишние скобки, во-вторых. Cогласно PEP8 nакая запись, хоть и допустима, но не принятf.
```suggestion
if a % 10 + a // 10 % 10 + a // 100 == b % 10 + b // 10 % 10 + b // 100:
    print('YES')
else:
    print('NO')
```